packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -253,6 +253,14 @@ export class DeltaManager
         return this.lastProcessedSequenceNumber;
     }
 
+    public get clientSequenceNumberObserved(): number {
```

Comment:
lastKnownClientSeqNumber to match below, and then not rename needed

Reply:
Removed and exposed the difference.